src/bluez/adapter.rs

```diff
@@ -162,6 +178,16 @@ async fn central_event(
             }
             _ => None,
         },
+        BluetoothEvent::Adapter {
+            id,
+            event: adapter_event,
+        } if id == adapter_id => match adapter_event {
+            AdapterEvent::Powered { powered } => {
+                let state = get_central_state(powered);
+                Some(CentralEvent::StateUpdate(state.into()))
```

Comment:
Why do you need the `.into()` here?

Reply:
i guess i dont need it :) fixed [bcf31a7](https://github.com/deviceplug/btleplug/pull/366/commits/bcf31a758619ffb71eeec2f0482a424e5ad9a40f)